src/Proj4.jl

```diff
@@ -46,19 +46,18 @@ end
 function _get_def(proj::Projection)
     @assert proj.rep != C_NULL
     opts = 0 # Apparently obsolete argument, not used in current proj source
-    bytestring(ccall((:pj_get_def, libproj), Cstring, (Ptr{Void}, Cint), proj.rep, opts))
+    bytestring(ccall((:pj_get_def, libproj), Compat.Cstring, (Ptr{Void}, Cint), proj.rep, opts))
 end
 
 """Low level interface to libproj transform, allowing user to specify strides"""
-function _transform!(src::Projection, dest::Projection, point_count, point_stride, x, y, z)
+function _transform!(src::Projection, dest::Projection, point_count::Clong,
+                     point_stride::Cint, x::Ptr{Cdouble}, y::Ptr{Cdouble}, z::Ptr{Cdouble})
```

Comment:
I'm still not sure that writing the C types is really necessary here for `point_count` and `point_stride`.  To me it seems just to be forcing you to do extra type casting down below.  According to the `ccall` documentation, there's an implicit call to `cconvert()` inserted for each argument passed to the `ccall` function - that would seem to completely take care of type conversion worries.